tests/scheduled_checks.py

```diff
@@ -0,0 +1,81 @@
+"""
+These tests are not called on every push or pull request. They're time-intensive, so on GitHub they're only called
+monthly. To run them locally, use the following command:
+
+    pytest -s -rpP tests/scheduled_checks.py
+
+Because the filename doesn't begin with test_, it's not automatically run with pytest. The -s flag is used to print
+the results of the tests, and the -rpP flags are used to suppress the output of the tests that pass.
+
+"""
+from astroquery.simbad import Simbad
+
+from astrodbkit2.utils import _name_formatter
+
+def test_SIMBAD_resolvable(db):
+    # Verify that all sources have valid coordinates
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Add all IDS to the Simbad output as well as the user-provided id
+    Simbad.add_votable_fields("ids")
+    Simbad.add_votable_fields("typed_id")
+
+    simbad_results = Simbad.query_objects(name_list)
+    duplicate_count = 0
+    for row in simbad_results[["TYPED_ID", "IDS"]].iterrows():
+        try:
+            name, ids = row[0].decode("utf-8"), row[1].decode("utf-8")
+        except AttributeError:
+            # Catch decoding error
+            name, ids = row[0], row[1]
+
+        simbad_names = [
+            _name_formatter(s)
+            for s in ids.split("|")
+            if _name_formatter(s) != "" and _name_formatter(s) is not None
+        ]
+
+        assert(
```

Comment:
One of the problems with having assertions inside a for-loop is that the first case that fails will break the test. If you have 100 sources and 50 of them fail this simbad check, you will only get notified for the very first one which means you'll spend a lot of time repeating the test to try and find all of them.

A better approach is to produce lists of names, both from the database and simbad, and then compare them in bulk. You can create a variable that stores a list of missing names and then assert that variable has a length of 0 (ie, no missing names) and if not- print it to get the full ist.